dpsg/docs/integrations/adyen/config.md

```diff
@@ -54,7 +54,7 @@ From their you can set up `Standard Notification` webhook.
 ### Webhook Config
 
 **Transport**
-- URL: https://dpsg.dev.deity.cloud/api/payment/webhook/{organization}/{project}/{environment}/adyen (replacing `{organization}`, `{project}` and `{environment}` with your details).
+- URL: https://dpsg.deity.cloud/api/payment/webhook/{organization}/{project}/{environment}/adyen (replacing `{organization}`, `{project}` and `{environment}` with your details).
```

Comment:
We have multiple potential regions, so this should be:
`https://dpsg.{region}.deity.cloud/api/payment/webhook/{organization}/{project}/{environment}/adyen`

And then mention that if it is `eu`, the region can be omitted.

Reply:
@jonas-at-deity - this is ready for re-review